tests/integration/test_trimmed_geometry.py

```diff
@@ -0,0 +1,246 @@
+            total_point_gaps = np.floor(edge.length.m * 1000 / points_between_corners)
+            current_gap = 0
+            while current_gap < total_point_gaps + 1:
+                p1 = edge.shape.evaluate_proportion(current_gap / total_point_gaps).position
+                uv = face.shape.project_point(p1).parameter
+                u, v = face.shape.get_proportional_parameters(uv)
+                normal = face.normal(u, v)
+                p2 = design.add_design_point("hair", p1 + normal / 800).value
+                create_sketch_line(design, p1, p2)
+                current_gap += 1
+    # Add isoparametric curves, not on linux
+    if modeler.client.backend_type != BackendType.LINUX_SERVICE:
+        param = 0.20
+        while param <= 1:
+            for face in body.faces:
+                face.create_isoparametric_curves(True, param)  # u
+                face.create_isoparametric_curves(False, param)  # v
+            param += 0.20
+    return design
+
+
+"""A fixture of the hedgehog design to test the surface and curve properties individually."""
+
+
+@pytest.fixture
+def hedgehog_design(modeler: Modeler):
+    h = create_hedgehog(modeler)
+    yield h
+
+
+"""Tests the surface properties for hedgehog"""
+
+
+def test_trimmed_surface_properties(hedgehog_design):
+    hedgehog_body = hedgehog_design.bodies[0]
+    faces = hedgehog_body.faces
+
+    expected_surface_properties = [
+        (
+            SurfaceType.SURFACETYPE_CYLINDER,
+            False,
+            TrimmedSurface,
+            (0.0, 6.283185307179586),
+            (0.0, 0.02),
+        ),
+        (
+            SurfaceType.SURFACETYPE_PLANE,
+            True,
+            ReversedTrimmedSurface,
+            (-0.02024, 0.02024),
+            (-0.02024, 0.02024),
+        ),
+        (
+            SurfaceType.SURFACETYPE_PLANE,
+            False,
+            TrimmedSurface,
+            (-0.02024, 0.02024),
+            (-0.02024, 0.02024),
+        ),
+        (
+            SurfaceType.SURFACETYPE_CYLINDER,
+            True,
+            ReversedTrimmedSurface,
+            (2.746801533890032, 6.677976426879348),
+            (0.0, 0.02),
+        ),
+        (
+            SurfaceType.SURFACETYPE_CYLINDER,
+            False,
+            TrimmedSurface,
+            (3.141592653589793, 6.283185307179586),
+            (0.0, 0.02),
+        ),
+    ]
+
+    for i, (surface_type, is_reversed, shape_type, interval_u, interval_v) in enumerate(
+        expected_surface_properties
+    ):
+        assert faces[i].surface_type == surface_type
+        assert faces[i]._is_reversed == is_reversed
+        assert isinstance(faces[i].shape, shape_type)
+        assert faces[i].shape.box_uv.interval_u == Interval(start=interval_u[0], end=interval_u[1])
+        assert faces[i].shape.box_uv.interval_v == Interval(start=interval_v[0], end=interval_v[1])
+
+
+"""Tests the normal vectors for hedgehog by using the ``BoxUV`` coordinates."""
```

Comment:
```suggestion
"""Tests the normal vectors for the hedgehog design using the BoxUV coordinates."""
```
None of the earlier occurrences to BoxUV were tagged as code entities, so I removed the double backticks.)